.coafile

```diff
@@ -12,12 +12,22 @@ use_spaces = True
 # will be executed sequentially; also we need the LineLengthBear to double
 # check the line length because PEP8Bear sometimes isn't able to correct the
 # linelength.
-bears = SpaceConsistencyBear, PyUnusedCodeBear, QuotesBear
+bears = SpaceConsistencyBear, QuotesBear
 language = Python
 preferred_quotation = '
 
 default_actions = *: ApplyPatchAction
 
+[python_imports]
```

Comment:
I would call it `flakes`, as that is closer to `autopep8` , and allows unused variables and other flakes to be included in this section.